### PR TITLE
 Use lambda expressions instead of anonymous classes in spring-aop tests

### PR DESCRIPTION
**Pull request by lijunyzzZ**
this PR for simplify some code

### CODE REVIEW

File: spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java
Comment (mentallurg) [inline spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java]: If optimize, then do one step further :)

> sort(advisors, OrderComparator.INSTANCE);

File: spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java
Comment (mentallurg) [inline spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java]: If optimize, then do one step further :)

> sort(advisors, OrderComparator.INSTANCE);

**Review by mentallurg — CHANGES_REQUESTED** [review]
It is fine except of 2 places with `sort`.

File: spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java
Comment (lijunyzzZ) [reply spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java]: Hello, I am modifying it to OrderComparator.sort(advisors);

File: spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java
Comment (lijunyzzZ) [reply spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java]: modify to OrderComparator.sort(advisors);

File: spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java
Comment (mentallurg) [inline spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java]: Good. Even better than I suggested :)

File: spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java
Comment (mentallurg) [inline spring-aop/src/test/java/org/springframework/aop/aspectj/annotation/AbstractAspectJAdvisorFactoryTests.java]: Good. Even better than I suggested :)

File: spring-aop/src/test/java/org/springframework/aop/support/PointcutsTests.java
Comment (mentallurg) [inline spring-aop/src/test/java/org/springframework/aop/support/PointcutsTests.java]: Many developers use "clazz" because "class" is a protected word. But other words like "type" could be better in such cases. How about following?

> return type -> type.equals(TestBean.class);

**Review by mentallurg — APPROVED** [review]
1. The usace of comparator is improved. OK.

2. "type" vs "clazz" is a matter of personal preferences. I can accept both.

Everything else if fine to me.

**Comment by mentallurg** [comment]
Please change you commit message. Use something that better describes your changes. Consider **"Use Java 8 streams instead of anonymous classes. Use Java 8 comparators."** or something similar.

File: spring-aop/src/test/java/org/springframework/aop/support/PointcutsTests.java
Comment (lijunyzzZ) [reply spring-aop/src/test/java/org/springframework/aop/support/PointcutsTests.java]: Thank you for your suggestion, I have modified it.

**Review by mentallurg — APPROVED** [review]

**Comment by sbrannen** [comment]
> mentallurg approved these changes yesterday

@mentallurg, when reviewing pull requests, please refrain from approving or rejecting a PR unless you are a core committer for the Spring Framework (see [CONTRIBUTING.md](https://github.com/spring-projects/spring-framework/blob/master/CONTRIBUTING.md#participate-in-reviews)).

Thanks!

File: spring-webflux/src/main/java/org/springframework/web/reactive/config/ViewResolverRegistry.java
Comment (sbrannen) [inline spring-webflux/src/main/java/org/springframework/web/reactive/config/ViewResolverRegistry.java]: For future reference, please do not delete lines such as this one, especially in files that you otherwise have not modified.

**Comment by sbrannen** [comment]
This has been merged into `master` in a0d50a546bc44ef0273bea46a80486a4025a958a.

Thanks!

Please note, however, that the PR [broke the build](https://build.spring.io/browse/SPR-PUB-7793) due to unused imports that violate Checkstyle rules. This has been addressed in d357ef706fca2cfcc970d51b5c582d58adb9cb0f, but please be cautious about this in future PRs.

**Pull request merged**
